### DISCUSSION TITLE
Measuring internal heat source

### DISCUSSION THREAD

**Discussion by Elch-HH** [firemodels/fds#13943] [Q&A]
Hi all,

I am currently using INTERNAL_HEAT_SOURCE in an obstruction to simulate air temperatures in a room with a heating obstruction.
I would like to measure somehow the heat that is produced by the obstrution so that i will know that it is the right value.
for example, if the obstuction volume is 0.5 m^3 and i defined the internal heat source to be 20kW/m^3, i would expect to measure 10 kW.
Is there a way to do that?

**Comment by Er9y714** [comment]
I do not have experience with this quantity but have you checked this verification case in Verification Guide? It might be helpful:
11.9.7 3-D Heat Diffusion in a Sphere (ht3d_sphere)

**Reply by Elch-HH** [reply]
Thanks for the reply.
I did look at various validation and verification cases including the sphere.
Unless I'm missing something, I don't think that the sphere case has a relevant measurement for my case.
